### DISCUSSION TITLE
SMS with 2FA code is sent twice

### DISCUSSION THREAD

**Discussion by JacquesSG** [community/community#64174] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

When logging in, I receive two SMS's with the same 2FA code. This started only a few days ago for me.

**Comment by arshil121** [comment, marked as answer]
it seems that receiving two SMS's with the same 2FA code is not a common issue. However, it is possible that the website or service you are using has a bug or glitch causing this issue. It is recommended to contact the customer support of the website or service and report the issue to them. They may be able to provide a solution or explanation for the issue. It is also important to note that SMS-based 2FA is not the most secure option for 2FA, as it is susceptible to SIM swapping attacks and interception by third parties

**Reply by nadiarahayufanny21-wq** [reply]
> it seems that receiving two SMS's with the same 2FA code is not a common issue. However, it is possible that the website or service you are using has a bug or glitch causing this issue. It is recommended to contact the customer support of the website or service and report the issue to them. They may be able to provide a solution or explanation for the issue. It is also important to note that SMS-based 2FA is not the most secure option for 2FA, as it is susceptible to SIM swapping attacks and interception by third parties

**Comment by shasjgasjhcb** [comment]
"Yakitori" is a Japanese term that refers to grilled skewered chicken, and "yakiniku" refers to grilled meat in general. [yaki nik in singapore ](=https://yakinikulike.org/)Singapore is known for its diverse culinary scene, and you can indeed find various types of Japanese restaurants that serve yakitori and yakiniku dishes

**Comment by marksoo90** [comment]
[Plaza Singapura](https://plaza-singapura.com/) adalah pusat perbelanjaan kontemporer yang terletak di sepanjang Orchard Road, Singapura, di sebelah stasiun MRT Dhoby Ghaut. Mal ini dikelola oleh CapitaLand dan dimiliki oleh CapitaMall Trust. Ada gerai ritel di atas tujuh lantai dan dua ruang bawah tanah.

**Comment by Rubel233** [comment]
Sms 2fa code recovery

**Comment by nguyenvanduong97qb-netizen** [comment]
Mở khóa Telegram

**Comment by nguyenvanduong97qb-netizen** [comment]
Mở khóa Telegram
